ObjectMapper.podspec

```diff
@@ -1,16 +1,21 @@
 Pod::Spec.new do |s|
   s.name = 'ObjectMapper'
-  s.version = '4.2.0'
+  s.version = '4.4.0'
   s.license = 'MIT'
+  # Ensure developers won't hit CocoaPods/CocoaPods#11402 with the resource
+  # bundle for the privacy manifest.
+  s.cocoapods_version = '>= 1.12.0'
   s.summary = 'JSON Object mapping written in Swift'
   s.homepage = 'https://github.com/tristanhimmelman/ObjectMapper'
   s.authors = { 'Tristan Himmelman' => 'tristanhimmelman@gmail.com' }
   s.source = { :git => 'https://github.com/tristanhimmelman/ObjectMapper.git', :tag => s.version.to_s }
-
-  s.watchos.deployment_target = '2.0'
-  s.ios.deployment_target = '13.0'
-  s.osx.deployment_target = '10.10'
-  s.tvos.deployment_target = '9.0'
+  s.resource_bundle = {
+    "Privacy" => "Sources/ObjectMapper/Resources/PrivacyInfo.xcprivacy"
+  }
+  s.watchos.deployment_target = '10.0'
+  s.ios.deployment_target = '17.0'
+  s.osx.deployment_target = '12.0'
+  s.tvos.deployment_target = '17.0'
```

Comment:
I assume "must be built with [Xcode 15](https://apps.apple.com/us/app/xcode/id497799835?mt=12) for [iOS 17](https://developer.apple.com/ios/)..." does not mean we should set the deployment target to iOS 17 as it will break most existing clients of the library who still might support earlier iOS versions.

If you check https://developer.apple.com/ios/submit/ it just says "must be built with a minimum of Xcode 15 and the iOS 17 SDK." There is no limitation to the deployment target it just has to be built with Xcode 15 and the latest SDK which makes sense.

I believe we will have to revert most of the introduced limitations in this pull request.

Reply:
@tristanhimmelman 
This is correct. Can you revert this code so the library can support old iOS versions? Thank you

Reply:
@mvn-thanhnguyen4-dn that makes sense. Can you submit a PR please

Reply:
@tristanhimmelman 
Please review this PR. Thank you
https://github.com/tristanhimmelman/ObjectMapper/pull/1146